File: 310971/Stankevich/jpa320/src/main/java/com/task/rest/model/Author.java
package com.task.rest.model;

import jakarta.persistence.*;
import lombok.*;

import java.util.List;

@Entity
@Table(name = "tbl_author")
@Data
@EqualsAndHashCode(callSuper = true)
@ToString(exclude = "tweets")
@NoArgsConstructor
@AllArgsConstructor
public class Author extends BaseEntity {

    @Column(nullable = false, length = 64, unique = true)
    private String login;

    @Column(nullable = false, length = 128)
    private String password;

    @Column(nullable = false, length = 64)
    private String firstname;

    @Column(nullable = false, length = 64)
    private String lastname;

    @OneToMany(mappedBy = "author", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Tweet> tweets;
}

File: 351004/Kuchko/ServerApp/ServerApp/Models/Entities/Author.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace ServerApp.Models.Entities;

[Table("tbl_author")]
public class Author : BaseEntity
{
    [Column("login")] public string Login { get; set; } = null!;

    [Column("password")] public string Password { get; set; } = null!;

    [Column("firstname")] public string Firstname { get; set; } = null!;

    [Column("lastname")] public string Lastname { get; set; } = null!;

    // связь 1 ко многим
    public virtual ICollection<Article> Articles { get; set; } = new List<Article>();
}